app/src/main/java/rick_and_morty/data/model/CharacterDetails.kt

```diff
@@ -0,0 +1,3 @@
+package rick_and_morty.data.model
+
+data class CharacterDetails(val title: String, val info: String)
```

Comment:
this is a type that is used exclusively in the UI layer, let's move it there

Reply:
done